rpm/libtiff.spec

```diff
@@ -49,7 +50,7 @@ Requires:  %{name} = %{version}-%{release}
 Man and info pages for %{name}.
 
 %prep
-%autosetup -n %{name}-%{version}/upstream
+%autosetup -p1 -n %{name}-%{version}/upstream
```

Comment:
-p1 is the default for `%autopatch` as used by `%autosetup`.
However it doesn't hurt to add it.

Reply:
> -p1 is the default for `%autopatch` as used by `%autosetup`. However it doesn't hurt to add it.

Are you sure? Because it doesn't work without it.

Reply:
We need to define -p1.